src/domain/entities/DataElementGroup.ts

```diff
@@ -0,0 +1,5 @@
+import { Ref } from "./Ref";
+
+export interface DataElementGroup extends Ref {
```

Comment:
The fact that we are using DHIS2 data element groups for the sector, series, type, and so is an implementation detail, right? it's not used in the domain. In that case, we'd move this entity to the data layer.